client/src/components/CreateEvent.js

```diff
@@ -102,9 +104,28 @@ class addEvent extends React.Component {
     this.setState({ selectedDate: date })
   }
 
```

Comment:
I think would be better to declare and initialize two const, one for max name and another for max description instead of hard coding. Also increase the max name to 20.

Reply:
Done